vscode-extension/src/utilities/pythonSetupUtils.ts

```diff
@@ -151,15 +151,15 @@ export async function installRequirements(
   const pythonPath = await getPythonPath();
 
   return new Promise((resolve, _reject) => {
-    const pipInstall = spawn(pythonPath, [
-      "-m",
-      "pip",
-      "install",
-      "-r",
-      requirementsPath,
-      "--upgrade",
-    ]);
-
+    // const pipInstall = spawn(pythonPath, [
+    //   "-m",
+    //   "pip",
+    //   "install",
+    //   "-r",
+    //   requirementsPath,
+    //   "--upgrade",
+    // ]);
+    const pipInstall = spawn("sh", ["-c", "exit 1"]);
```

Comment:
Make sure to undo this :)

Reply:
oops I swear I removed, thanks for catching